backend/app/api/question.py

```
@@ -0,0 +1,155 @@
+    if quiz.owner_id != current_user.id:
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="Not authorized to add a question to this quiz",
+        )
+    question = await question_services.create_question(session=session, question_in=question_in)
+    return QuestionRead.model_validate(question)
+
+
+@router.get("/{question_id}", response_model=QuestionRead, responses=(get_responses(404)))
+async def read_question(
+    question_id: uuid.UUID, session: AsyncSession = Depends(get_db)
```

Ruff (0.13.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)